src/utils/api/fetchWrapperPieces.ts

```diff
@@ -0,0 +1,24 @@
+import type {
+  HTTPClient,
+  HTTPClientOptions,
+} from '@folio/stripes-acq-components';
+
+import { ORDER_WRAPPER_PIECES_API } from '../../constants';
+
+interface WrapperPiece {
+    piece: ACQ.Piece;
+    poLine: ACQ.OrderLine;
+    purchaseOrder: ACQ.Order;
+    title: ACQ.Title;
+    vendorId: string;
+}
+
+interface WrapperDataShape {
+  wrapperPieces: WrapperPiece[];
+  totalRecords: number;
+}
+
+export const fetchWrapperPieces =
+(httpClient: HTTPClient) => async (options: HTTPClientOptions): Promise<WrapperDataShape> => {
```

Comment:
Please don't break fn expression into new line. If expression is too long (linter warnings), use `return` for the second fn